internal/publish/preflight_checks.go

```diff
@@ -30,17 +29,12 @@ func (p *defaultPublisher) checkConfiguration(client connect.APIClient) error {
 	}
 	log.Info("Publishing with credentials", "username", user.Username, "email", user.Email)
 
+	var existingContentID *types.ContentID
 	if p.Target != nil {
```

Comment:
Removed the code below as it was simply checking if the last content type deployed (from the content record) is the same as the "new" value within the configuration file. Since we really want to be checking against the Connect server's state for this deployment, this is being removed. It will be replaced in a PR coming up for issue #2301.